### PR TITLE
type ahead directive was added to create play modal window

### PR DESCRIPTION
**Pull request by andrewMuntyan**

### CODE REVIEW

File: client/static/app/modules/games/directives/type-ahead/type-ahead.directive.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+define(function(require) {
+    'use strict';
+
+    var typeAheadTemplate = require('text!./type-ahead.tpl.html'),
+        controller = require('../../controllers/type-ahead.controller');
```
Comment (artyomtrityak) [inline client/static/app/modules/games/directives/type-ahead/type-ahead.directive.js]: no need to create type-ahead subfolder

File: client/static/app/modules/games/main.js [hunk 2]
```diff
@@ -11,10 +11,13 @@ define(function(require) {
       GamesDetailsTpl = require('text!./templates/game-details.tpl.html'),
 
       GameService = require('./services/game.service'),
-      
+
+      TypeAheadDirective = require('./directives/type-ahead/type-ahead.directive'),
```
Comment (artyomtrityak) [inline client/static/app/modules/games/main.js]: remove type-ahead subfolder

File: client/static/assets/scss/ui-select.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.type-ahead {
```
Comment (artyomtrityak) [inline client/static/assets/scss/ui-select.scss]: we are using BEM styles

**Comment by artyomtrityak** [comment]
good job, remove type-ahead subfolder and commit

File: client/static/app/modules/games/controllers/type-ahead.controller.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+                vm.games = data;
+            });
+        }
+
+        function gameSelect(item) {
+            vm.playDataName = item.nameOrigin;
+        }
+    }
+});
+
```
Comment (Vasso) [inline client/static/app/modules/games/controllers/type-ahead.controller.js]: wrong file indentation

File: client/static/app/modules/games/directives/type-ahead/type-ahead.directive.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+            template: typeAheadTemplate,
+            controller: controller,
+            controllerAs: 'dgTypeAheadIns',
+            bindToController: true,
+            scope: {
+                playDataName: '='
+            }
+        };
+    }
+});
```
Comment (Vasso) [inline client/static/app/modules/games/directives/type-ahead/type-ahead.directive.js]: wrong indentation

**Pull request merged**